src/main/java/methods/Setup.java

```diff
@@ -61,12 +61,13 @@ public static void init(File file) throws IOException
         try(FileWriter fw = new FileWriter(file))
         {
             fw.write(String.format(
-                "lastfm_apikey=%s\n" +
+                    "lastfm_apikey=%s\n" +
                     "lastfm_secret=%s\n" +
                     "lastfm_username=%s\n" +
                     "lastfm_password=%s\n" +
-                    "clonehero_data_folder=%s",
-                apiKey, secret, user, pass, dataFolder));
+                    "clonehero_data_folder=%s\n" +
+                    "scrobble_threshold_seconds=%s25",
```

Comment:
The `%s` format specifier needs to be removed. The value should be changed to either `30` or the variable in the `Statics` class (if you decide to do that).

Reply:
I misunderstood what %s does. I have added Statics.DEFAULT_SCROBBLE_THRESHOLD. I think it makes sense for the default value to be in the default settings file, so I left %s (and passed the corresponding argument to String.format).